src/ep.config.json

```diff
@@ -54,7 +54,7 @@
   "gaTrackingId": "",
   "arKit": {
     "enable": true,
-    "skuArImagesUrl": "https://s3.amazonaws.com/referenceexp/ar/%sku%.usdz"
+    "skuArImagesUrl": "https://s3.amazonaws.com/referenceexp/ar/%sku%.usdz#callToAction=Add%20to%20cart&checkoutTitle=%title%&checkoutSubtitle=%description%&price=%price%"
```

Comment:
Since the descriptions get cut off, it may be helpful to make the box larger using `customHeight=large`